src/aiperf/common/config/image_config.py

```
@@ -131,4 +138,38 @@ class ImageConfig(BaseConfig):
+    @model_validator(mode="after")
+    def _validate_image_options(self) -> Self:
+        """Validate the image options."""
+        image_options_set = {
+            *self.model_fields_set,
+            *self.width.model_fields_set,
+            *self.height.model_fields_set,
+        }
+        if not self.images_enabled() and image_options_set:
+            raise ValueError(
+                "Image generation is disabled but image options were provided. Please set `--image-width-mean` and `--image-height-mean` to enable image generation."
+            )
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)